src/services/categories_service/categories_service.ts

```diff
@@ -0,0 +1,5 @@
+import { EngagementCategory } from "../../schemas/engagement_category";
+
+export abstract class CategoriesService {
```

Comment:
Can we rename all instances of `CategoriesService` to `CategoryService`, including the implementations?